CONFIGDOC.md

```diff
@@ -22,68 +23,72 @@ When you define an automated pipeline of tasks in Jenkins&trade;, whether in the
 
 When you specify a MATLAB version in the **Build Environment** section, the plugin prepends its `bin` folder to the system PATH environment variable of the build agent and invokes it to perform the build. If the build agent already has your desired MATLAB on the path, then you are not required to select **Use MATLAB version**. The plugin always uses the topmost MATLAB version on the system path. The build fails if the operating system cannot find MATLAB on the path.
 
-**Note:** The [`matlabroot`](https://www.mathworks.com/help/matlab/ref/matlabroot.html) function returns the full path to the MATLAB root folder, which depends on the platform, MATLAB version, and installation location. This table shows examples of the root folder path on different platforms. 
+You can use the [`matlabroot`](https://www.mathworks.com/help/matlab/ref/matlabroot.html) function to return the full path to your desired MATLAB root folder. The path depends on the platform, MATLAB version, and installation location. This table shows examples of the root folder path on different platforms. 
 
 | Platform     | Path to MATLAB Root Folder      |
 |--------------|---------------------------------|
-| Windows&reg; | C:\Program Files\MATLAB\R2020b  |
-| Linux&reg;   | /usr/local/MATLAB/R2020b        |
-| macOS        | /Applications/MATLAB_R2020b.app |
+| Windows      | C:\Program Files\MATLAB\R2021a  |
+| Linux&reg;   | /usr/local/MATLAB/R2021a        |
+| macOS        | /Applications/MATLAB_R2021a.app |
 
 ### Specify Build Steps
 When you set up the **Build** section of the project configuration window, the plugin provides you with the **Run MATLAB Command** and **Run MATLAB Tests** build steps.
 
 If you use a source code management (SCM) system such as Git&trade;, then your project should include the appropriate SCM configuration to check out the code before it can invoke the plugin. If you do not use any SCM systems to manage your code, then an additional build step might be required to ensure that the code is available in the project workspace before the build starts.
 
 #### Run MATLAB Command
-The **Run MATLAB Command** build step lets you specify MATLAB commands tailored to your specific needs. For example, you can use this build step to customize your test run or add a MATLAB related step to your automated pipeline of tasks.
+The **Run MATLAB Command** build step lets you run MATLAB scripts, functions, and statements. You can use this build step to customize your test run or execute any MATLAB commands.
 
-If you specify more than one MATLAB command, use a comma or semicolon to separate the commands. The build fails if the execution of any command results in an error.
+Specify the MATLAB script, function, or statement you want to execute in the **Command** box. If you specify more than one script, function, or statement, use a comma or semicolon to separate them. If you want to run a script or function, do not specify the file extension.

```

Comment:
Would this paragraph be more intuitive if it mentioned how you enter in commands/scripts/functions as though you're calling it within the MATLAB Command Window? 🤔

Reply:
I agree, probably mentioning specify commands like you do for MATLAB command window may be slightly more intuitive.

Reply:
I am not so sure if this would be necessary, because the screenshot should be clarifying 😋.